gwpy/cli/qtransform.py

```diff
@@ -112,12 +112,23 @@ def _finalize_arguments(self, args):
         """
         gps = args.gps
         search = args.search
-        # ennsure we have enough data for filter settling
+        # ensure we have enough data for filter settling
         max_plot = max(args.plot)
         search = max(search, max_plot * 2 + 6)
+        args.search = search
         self.log(3, "Search window: {0:.0f} sec, max plot window {1:.0f}".
                  format(search, max_plot))
 
+        # make sure we don't create too big interpolations
+        xpix = 1200.
```

Comment:
@areeda, it might reduce cognitive complexity to instead do the check like this:

```python
try:
    m = re.match('(\d+)x(\d+)', args.geometry)
    xpix = float(m.group(1))
except:
    xpix = 1200.
```
This way, there's only one decision point as opposed to two.

Reply:
I really don't get 'cognitive complexity' but thank for fixing that.